index.html

```diff
@@ -568,6 +568,24 @@ <h1>The Registry</h1>
         </td>
     </tr>
 
+    <tr>
+        <td>
+            did:https:
```

Comment:
Ooof, this will clearly need to be discussed. At best, it's misguided and we should use another protocol to bind DID Documents to resources published via the HTTPS protocol. At worst, it'll be viewed as a private company (ConsenSys/uPort) trying to use the DID work to squat on a very public namespace (https).

There are probably other ways to address whatever problem this DID Method is trying to solve. We should discuss at RWoT.

/cc @burnburn @chaals @prototypo

Reply:
Also /cc @ChristopherA @kimdhamilton @jandrieu